html-boilerplate/css/index.css

```
@@ -1,26 +1,88 @@
 
-body {background-color: powderblue;
-      font-family: verdana;
+body {
+  background-color: rgb(0, 0, 228);
+  font-family: verdana; 
+  display: flex;
+  flex-direction: column;
+  margin: 24px;
 }	
-p {font-family: "Sofia", sans-serif; font-size: 20px; margin: 5px; padding: 5px;}
-h1 {color: red; font-size: 30px;}
-h2 {color: royalblue; font-size: 30px;}
-ul {color: purple;}
+
+p {font-family: "Sofia", sans-serif; 
+  font-size: 20px;
+  margin: 5px; 
+  padding: 5px;}
+
+h1 {color: peachpuff; font-size: 30px;}
+h2 {color: peachpuff; font-size: 30px;}
+ul {color: black;}
+
 section {margin: 5px; padding: 5px;}
-div {
-    font-family: "Sofia", sans-serif; 
+
+div {font-family: "Sofia", sans-serif; 
     padding: 10px;
     text-align: center;
     margin-bottom: 5px;
-    display: inline-block;
-  }
-  label{color: orangered; font-size: 25px;}
-  input, textarea{color: green; font-size: 20px; padding: 10px;border: blue;}
-  label.name{margin: 15px; padding: 10px;}
-  button{background-color: rgb(19, 175, 214); padding: 20px; margin: 20px}
-  .messages{font-family: "Sofia", sans-serif; 
+    display: inline-block;}
+
+label{color: peachpuff; 
+  font-size: 25px;}
+
+  input, textarea{color: green; 
+    font-size: 20px; 
+    padding: 10px;
+    border: blue;}
+  
+label.name{margin: 15px; padding: 10px;}
+  
+button{background-color: rgb(19, 175, 214);
+     padding: 10px;
+     margin: 15px}
+  
+.messages.h2{font-family: "Sofia", sans-serif; 
+    color: peachpuff;
     padding: 10px;
     text-align: right;
     margin-bottom: 5px;
     display: inline-block}
-footer{background-color: rgb(214, 19, 139); padding: 20px; margin: 20px; text-align: center;}
\ No newline at end of file
+
+footer{background-color: orangered; 
+  padding: 5px; 
+  margin: 5px;
+  text-align: center;}
+
+a{color: peachpuff;}
+p{color: peachpuff;}
+
+.navbar a {
+  float: right;
+  display: block;
+  color: peachpuff;
+  text-align: right;
+  padding: 15px 18px;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+ul {
+  background-color: orangered;
+  padding: 16px;
+  border: 4px solid blue;
+  gap: 8px;
+  display: flex;
+}
+
+li {
+  background: peachpuff;
+  border: 4px solid blue;
+  height: 48px;
+  flex: 1;
+  padding: 4px;
+}
+
+form {display: grid; grid-template-columns: 1;}
+
+@media (max-width: 600px) {
+  body {
+    margin: 8px;
+  }
```

At smaller widths, some of the boxes in the skills and experience sections get chopped off on the right side, 
so let's stack the boxes vertically when on a smaller screen
```suggestion
  ul {
    flex-flow: column;
    }
  }
```